### DISCUSSION TITLE
WinUI + Pharmacist (ReactiveUI Events) 🤔

### DISCUSSION THREAD

**Discussion by bojan-sala-igt-com** [reactiveui/ReactiveUI#3664] [Q&A] [answered]
Pharmacist doesn't seem to be compatible with WinUI / UWP... Does anyone know how should one go about using `this.Events()` pattern in WinUI as we can do in, say, WPF?

Unrelated, but, I noticed that there was a same problem with Avalonia - I just couldn't get it to work. Has anyone had any luck with either of these?

We can still use `Observable.FromEventPattern(obj, "EventName")`, but I was just wondering if that's now officially what we do now or I was just too dumb to set up Pharmacist successfuly.

**Comment by glennawatson** [comment, marked as answer]
We mostly moved on from Pharmacist and now using https://github.com/reactivemarbles/observableevents

**Reply by bojan-sala-igt-com** [reply]
I had no idea this existed! Thanks a bunch 👌.
